### DISCUSSION TITLE
3D Position hold and stick movements

### DISCUSSION THREAD

**Discussion by Berlerd** [iNavFlight/inav#7134] [General]
Hi All,
I have a tyro 129 quad, working perfect in Acro and angle.
setting the quad into 3D position hold, it will stay in one place moving slightly.
If I move the sticks to any direction the quad will bank to the left and drift away with no relation to the stick movement.
releasing the sticks will cause the quad to hold its 3D location again.

Expecting to have stick movements in 3D position hold to allow relocating the quad in new place. 
What did I miss in my configuration?

Regards,
Danny

**Comment by JulioCesarMatias** [comment]
switch autopilot mode from `Attitude` to `Cruise` in `Advanced Tunning`
